<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Random_Point"/>
   <addaction name="actionAdd_Random_Box"/>
   <addaction name="separator"/>
   <addaction name="actionSet_Box_Selection_Mode"/>
   <addaction name="actionSet_Nearest_Selection_Mode"/>
   <addaction name="separator"/>
   <addaction name="actionFind_only_Boxes"/>
   <addaction name="actionFind_only_Points"/>
   <addaction name="separator"/>
   <addaction name="actionClear_Selection"/>
  </widget>
  <action name="actionAdd_Random_Point">
   <property name="text">
    <string>Add Random Point</string>
   </property>
  </action>
  <action name="actionAdd_Random_Box">
   <property name="text">
    <string>Add Random Box</string>
   </property>
  </action>
  <action name="actionSet_Box_Selection_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set Box Selection Mode</string>
   </property>
  </action>
  <action name="actionSet_Nearest_Selection_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set Nearest Selection Mode</string>
   </property>
  </action>
  <action name="actionClear_Selection">
   <property name="text">
    <string>Clear Selection</string>
   </property>
  </action>
  <action name="actionFind_only_Boxes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Find only Boxes</string>
   </property>
  </action>
  <action name="actionFind_only_Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Find only Points</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
